src/Illuminate/Concurrency/ProcessDriver.php

```diff
@@ -25,13 +25,18 @@ public function __construct(protected ProcessFactory $processFactory)
      */
     public function run(Closure|array $tasks): array
     {
-        $php = (new PhpExecutableFinder)->find(false);
+        $php = Application::phpBinary();
```

Comment:
This fixes issues caused by the binary path via Herd: `/Users/<user>/Library/Application Support/Herd/bin//php`, but I'm wondering if `Application` can be used here since Concurrency doesn't depend on it in `composer.json`?